### ISSUE TITLE
Blitter crash

### ISSUE THREAD

**Issue by dirkwhoffmann** [dirkwhoffmann/vAmiga#205]
Blitter can crash when switching from level 0 to level 2 when the emulator is running.

<img width="1410" alt="Bildschirmfoto 2019-10-25 um 10 06 04" src="https://user-images.githubusercontent.com/12561945/67554202-30fd7d80-f70f-11e9-9e44-fe27cacfdcc6.png">

**Comment by dirkwhoffmann** [comment]
Presumably fixed in vAmiga 0.43

Activity monitoring has been implemented in this release, too: 

<img width="720" alt="Bildschirmfoto 2019-10-25 um 17 08 28" src="https://user-images.githubusercontent.com/12561945/67584710-7ee5a600-f74e-11e9-90ca-f8b709813652.png">

**Comment by mithrendal** [comment]
My new love is this activity monitor thing. It is fun to look at demos or games and to learn how they are using the Amiga.  And it looks so nice.😍

While playing with it, I did asked me why can't we display the CPU activity as well, like in the activity monitor of macos?

Then I instantly rejected my question because I thought that it is maybe not possible to let the old 68k CPU do nothing.🥴 But I remembered that there was a special 68k command. 

```
NOP
Execution continues with the instruction following the NOP instruction. The NOP instruction does not begin execution until all pending bus cycles are completed.
```

Count all commands as activity except NOP ? My guess is that the CPU would always show 100%.😫🤪

What the heck is the macos CPU activity monitor showing us ? Maybe it counts only the bus activity?  🤨 

Then whenever the 68k-CPU accesses memory then the there is CPU (bus)-activity, we could count that. Or maybe it is even better to count only CPU writes to the memory as activity? This way we could blame games which are moving graphical objects with CPU instead of the blitter.🙃
